src/misc/ssEconomy.lua

```diff
@@ -180,6 +184,8 @@ function ssEconomy:loadGameFinished()
     -- Some calculations to make the code faster on the hotpath
     ssEconomy.aiPricePerMSWork = ssEconomy.aiPricePerHourWork / (60 * 60 * 1000)
     ssEconomy.aiPricePerMSOverwork = ssEconomy.aiPricePerHourOverwork / (60 * 60 * 1000)
+    local vanillaPricePerHour = 2000
+    ssEconomy.aiVanillaToSeasonsMultiplier = ssEconomy.aiPricePerHourWork / vanillaPricePerHour
```

Comment:
Oh i see. That is old.

Better is to use a 'constant':
`ssEconomy.VANILLA_AI_PER_HOUR = 2000` And then use that.

I'll update that document